SpatialGDK/Source/SpatialGDK/Private/Interop/RPCExecutor.cpp

```diff
@@ -47,10 +49,22 @@ bool RPCExecutor::ExecuteCommand(const FCrossServerRPCParams& Params)
 	const USpatialGDKSettings* SpatialSettings = GetDefault<USpatialGDKSettings>();
 
 	const float TimeQueued = (FDateTime::Now() - Params.Timestamp).GetTotalSeconds();
-	bool CanProcessRPC = UnresolvedRefs.Num() == 0 || SpatialSettings->QueuedIncomingRPCWaitTime < TimeQueued;
-
+	bool CanProcessRPC = UnresolvedRefs.Num() == 0 || SpatialSettings->QueuedIncomingRPCWaitTime < TimeQueued
+						 || (Function->SpatialFunctionFlags & SPATIALFUNC_NeverQueueForUnresolvedParameters) != 0;
 	if (CanProcessRPC)
 	{
+		if (UnresolvedRefs.Num() > 0 && SpatialSettings->QueuedIncomingRPCWaitTime < TimeQueued
+			&& (Function->SpatialFunctionFlags & SPATIALFUNC_AllowUnresolvedParameters) == 0)
```

Comment:
Why aren't you checking for `SPATIALFUNC_NeverQueueForUnresolvedParameters` as well here?